test/util.ts

```diff
@@ -1,6 +1,23 @@
+import * as fs from 'fs';
+
+import { IOptions } from '../src';
+
+const rootCertificate = fs.readFileSync(`${__dirname}/certs/certs/ca.crt`);
+
 /**
- * Returns etcd hosts to test against.
+ * Returns the host to test against.
  */
-export function getHosts(): string {
+export function getHost(): string {
   return process.env.ETCD_ADDR || '127.0.0.1:2379';
 }
+
+/**
+ * Returns etcd options to use for connections.
+ */
+export function getOptions(defaults: Partial<IOptions> = {}): IOptions {
+  return {
+    hosts: getHost(),
+    credentials: { rootCertificate },
+    ...defaults,
```

Comment:
Yep! That's intended. In some cases we purposely pass invalid hosts in and want them to override.